app/cafe.py

```diff
@@ -0,0 +1,26 @@
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+from datetime import date
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        count = 0
+        if "vaccine" not in visitor:
+            count += 1
+            raise NotVaccinatedError("NotVaccinatedError")
+
+        if visitor["vaccine"]["expiration_date"] < date.today():
+            count += 1
+            raise OutdatedVaccineError("OutdatedVaccineError")
+
+        if visitor["wearing_a_mask"] is False:
+            count += 1
+            raise NotWearingMaskError("NotWearingMaskError")

```

Comment:
The error message for `NotWearingMaskError` is not descriptive. The checklist suggests using descriptive error messages when raising exceptions.